### PR TITLE
fix request path;

### PR DESCRIPTION
**Pull request by realer01**

### CODE REVIEW

File: aiohttp_debugtoolbar/panels/request_vars.py [hunk 1]
```diff
@@ -32,10 +32,13 @@ def process_response(self, response):
...
-        # and maybe aiohttp_security
 
-        # if hasattr(request, 'session'):
-        #     data.update({
-        #         'session': dictrepr(request.session),
-        #     })
+        # TODO: think about aiohttp_security
+
+        # session to separate table
+        if hasattr(request, 'session'):
```
Comment (asvetlov) [inline aiohttp_debugtoolbar/panels/request_vars.py]: Should be `request.get('aiohttp_session')`

Comment (asvetlov) [inline aiohttp_debugtoolbar/panels/request_vars.py]: Could you add a unit test?

File: aiohttp_debugtoolbar/panels/request_vars.py [hunk 2]
```diff
@@ -32,10 +32,13 @@ def process_response(self, response):
...
+            session = request.session
+            if not session.empty:
+                data.update({
+                    'session': [(k, session[k]) for k in session],
```
Comment (asvetlov) [inline aiohttp_debugtoolbar/panels/request_vars.py]: Use sorted keys: `for k in sorted(session)`.

File: aiohttp_debugtoolbar/panels/request_vars.py (hunk 1 above)
Comment (realer01) [inline aiohttp_debugtoolbar/panels/request_vars.py]: If you show me test example.

File: aiohttp_debugtoolbar/panels/request_vars.py (hunk 2 above)
Comment (realer01) [inline aiohttp_debugtoolbar/panels/request_vars.py]: fixed

**Comment by asvetlov** [comment]
Thanks

**Pull request merged**